src/index.js

```diff
@@ -0,0 +1,154 @@
+const abind = require('abind');
+const AWS = require('aws-sdk');
+const debug = require('debug')('aws-transcode');
+const delay = require('delay');
+
+class AwsTranscoder {
+  /**
+   * Constructor.
+   *
+   * @param {string} config.checkExistsInBucket (optional) if set we will check if an output already
```

Comment:
I believe the correct way to indicate optional params is like this:
```suggestion
   * @param {string} [config.checkExistsInBucket] if set we will check if an output already
```